package.json

```diff
@@ -9,20 +9,22 @@
     "src"
   ],
   "scripts": {
+    "build": "tsc --sourceMap",
+    "changelog": "conventional-changelog -p angular -i CHANGELOG.md -w",
+    "clean": "rimraf dist",
+    "coverage": "nyc --reporter lcov ava",
     "#postinstall": "Remove bundled typings to use patched version without Promise support",
     "postinstall": "rimraf node_modules/ava/index.d.ts",
-    "clean": "rimraf dist",
+    "postversion": "git push && git push --tags",
     "prebuild": "npm run clean",
-    "build": "tsc --sourceMap",
+    "precoverage": "npm run clean && tsc --inlineSourceMap",
+    "prepublish": "not-in-install && npm run build || echo 'In install cycle, skipping prepublish'",
     "pretest": "npm run build",
-    "test": "ava",
-    "changelog": "conventional-changelog -p angular -i CHANGELOG.md -w",
     "preversion": "npm test",
+    "start": "npm run watch",
+    "test": "ava",
     "version": "npm run build && conventional-changelog -p angular -v -r 0 > CHANGELOG.md && git add -A .",
-    "postversion": "git push && git push --tags",
-    "prepublish": "not-in-install && npm run build || echo 'In install cycle, skipping prepublish'",
-    "precoverage": "npm run clean && tsc --inlineSourceMap",
-    "coverage": "nyc --reporter lcov ava"
+    "watch": "npm run test -- --watch"
```

Comment:
This is not sufficent. Devs would expect to start a full watch process. This only watches the tests.
You still need to build the sources (and probably what the build process).